src/Annotation/Type.php

```diff
@@ -14,43 +14,35 @@ class Type implements Annotation
 {
     /**
      * Type name.
-     *
-     * @var string
      */
-    public $name;
+    public string $name;
 
     /**
      * Type inherited interfaces.
      *
      * @var string[]
      */
-    public $interfaces;
+    public array $interfaces;
```

Comment:
same here

Reply:
I didn't understand, what is wrong here?

Reply:
Wrong diff in the first place, you can forget it.